### DISCUSSION TITLE
Copilot stopped connecting to the account

### DISCUSSION THREAD

**Discussion by skibaa** [community/community#19613] [Copilot Conversations]
Until recently copilot worked for me very well and now it's hard to code without it, thanks to all developers. I use it in IntelliJ for Kotlin most often

But during the last week it stopped being able to authorize with github website. I press "Login to GitHub", it generates a code, I press "copy and open", and paste 8 letter code on GitHub in Chrome. After a short while GitHub sites says I'm authorized, but IntelliJ is keeping waiting and finally reports it's unable to authorize, so it stays disabled. Screenshots attached
![Screen Shot 2022-06-28 at 14 56 48](https://user-images.githubusercontent.com/194390/176175189-e2905bb5-7755-414f-8b82-49e2fb571a80.png)
![Screen Shot 2022-06-28 at 14 56 38](https://user-images.githubusercontent.com/194390/176175212-84bcc7a1-3317-4317-82f5-f982ad123317.png)

BTW this link https://github.com/features/copilot/signup gives 404 error

**Comment by davidlacho** [comment]
I'm having the same issue and I haven't found a resolution yet.

**Comment by manasesjesus** [comment]
I had a similar issue and got fixed after allowing Copilot in my profile settings. Hope this helps.

<img width="936" alt="githubcop" src="https://user-images.githubusercontent.com/24204142/177178814-a38f5fb1-452b-4d1a-8ced-33c5502e052a.png">
